.github/workflows/ci.yml

```
@@ -1,91 +1,162 @@
+    
+    - name: Check UI formatting
+      run: make format-check-ui
+    
+    - name: Check TypeScript compilation
+      run: make check
+    
+    - name: Run UI tests
+      run: make test-ui
+    
+    - name: Upload UI coverage to Codecov
+      uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)